src/js/device-state.js

```diff
@@ -84,11 +93,16 @@ export default class DeviceStateObserver {
   }
 
   getDeviceState(content) {
+    if (typeof content !== 'string') {
```

Comment:
Seems this bug was introduce by the r[efactoring 3 months ago](https://github.com/axa-ch/patterns-library/commit/61d7cafafdb43beb64f5326343e6aef215ee1e23) 🤔 
May I ask to restore the original script?